src/ILLink.Tasks/FindNativeDeps.cs

```diff
@@ -53,26 +53,25 @@ public override bool Execute ()
 
 			var managedAssemblies = ManagedAssemblyPaths.Select (i => i.ItemSpec).ToArray ();
 			foreach (string managedAssembly in managedAssemblies) {
-				using (var peReader = new PEReader(new FileStream (managedAssembly, FileMode.Open, FileAccess.Read, FileShare.Read))) {
-					if (peReader.HasMetadata) {
-						var reader = peReader.GetMetadataReader ();
-						for (int i = 1, count = reader.GetTableRowCount (TableIndex.ModuleRef); i <= count; i++) {
-							var moduleRef = reader.GetModuleReference (MetadataTokens.ModuleReferenceHandle (i));
-							var moduleName = reader.GetString (moduleRef.Name);
+				using var peReader = new PEReader(new FileStream (managedAssembly, FileMode.Open, FileAccess.Read, FileShare.Read));
```

Comment:
I am really not sure this using syntax is better. It really hides the scope where the file can be released.